server/routes/label.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const fs = require('fs');
+const rimraf = require('rimraf')
+
+
+const router = new express.Router();
+
+const defaultLocation = '..'
+
+router.post('/createLabel', (req, res) => {
```

Comment:
Update the endpoint here to `/create` in accordance with the `route.use` endpoints that I mentioned above.